modules/KoreBuild.Tasks/Internal/ProjectModel/ProjectInfo.cs

```diff
@@ -44,7 +41,5 @@ internal class ProjectInfo
 
         public IReadOnlyList<ProjectFrameworkInfo> Frameworks { get; }
         public IReadOnlyList<DotNetCliReferenceInfo> Tools { get; }
-
-        public MSBuildProjectExtension TargetsExtension { get; }
```

Comment:
I guess we just didn't use this anywhere?

Reply:
Correct. It was only used by lineups.